tools/oio-bootstrap.py

```diff
@@ -426,6 +426,32 @@ def save_coverage():
     - {type: system}
 """
 
+template_beanstalk_watch = """
+host: ${IP}
+port: ${PORT}
+type: beanstalkd
+location: ${LOC}
+checks:
+    - {type: beanstalk, tube: oio}
+slots:
+    - beanstalk
+stats:
+    - {type: system}
```

Comment:
Beanstalkd services store data on the file system, thus we should monitor the volume they are using.
```suggestion
    - {type: system}
    - {type: volume, path: ${VOLUME}}
```

Reply:
Done